
{'date': 'Tue Sep 03 2019 09:38:53.634 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 12
  templo = Cena(DI["CENA"])
  ^
IndentationError: expected an indented block
'''},
{'date': 'Tue Sep 03 2019 09:40:40.755 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 22
  if__name__ == "__main__":
                           ^
SyntaxError: invalid syntax
'''},
{'date': 'Tue Sep 03 2019 09:44:29.840 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 22
  if__name__ == "__main__":
                           ^
SyntaxError: invalid syntax
'''},
{'date': 'Tue Sep 03 2019 09:47:45.110 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 21
  if__name__ == "__main__":
                           ^
SyntaxError: invalid syntax
'''},
{'date': 'Tue Sep 03 2019 09:50:30.689 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 3
    from stacy.main import fogo
ImportError: cannot import name 'fogo'

ImportError
Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 264
    action()
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 3
    from stacy.main import fogo
'''},
{'date': 'Tue Sep 03 2019 09:51:45.448 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 21
    inca.inicia() 
  module <module> line 14
    tesouro.direita = fogo
NameError: name 'fogo' is not defined
'''},
{'date': 'Tue Sep 03 2019 10:23:04.867 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 35
  inca = Jogo()  # Inca()
  ^
IndentationError: expected an indented block
'''},
{'date': 'Tue Sep 03 2019 10:27:43.52 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 35
  inca = Jogo()  # Inca()
  ^
IndentationError: expected an indented block
'''},
{'date': 'Tue Sep 03 2019 10:29:04.88 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 34
  inca = Jogo()  # Inca()
  ^
IndentationError: expected an indented block
'''},
{'date': 'Tue Sep 03 2019 10:30:09.606 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 35
    inca = Jogo()  # Inca()
  module <module> line 31
    self.templo = Cena(DI["TEMPLO"])
KeyError: TEMPLO
'''},
{'date': 'Tue Sep 03 2019 10:52:26.225 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 42
    inca = Jogo()  # Inca()
  module <module> line 35
    self.baralho = Carta().baralho()
  module <module> line 29
    self.carta = Cena(DI["TESOURO"])
KeyError: TESOURO
'''},
{'date': 'Tue Sep 03 2019 10:58:15.986 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 42
    inca = Jogo()  # Inca()
  module <module> line 35
    self.baralho = Carta().baralho()
  module <module> line 29
    self.cartas = [Cena(RDI[uma_imagem]) for uma_imagem in IMAGENS]
NameError: name 'RDI' is not defined
'''},
{'date': 'Tue Sep 03 2019 11:22:37.667 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 21
  pedras preciosas = Cena(DI["PEDRAS PRECIOSAS"])
          ^
SyntaxError: invalid syntax
'''},
{'date': 'Sun Sep 08 2019 23:42:28.419 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 21
  pedras preciosas = Cena(DI["PEDRAS PRECIOSAS"])
          ^
SyntaxError: invalid syntax
'''},
{'date': 'Mon Sep 09 2019 00:19:08.255 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 4
    from random import shuffle,STYLE
ImportError: cannot import name 'STYLE'

ImportError
Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 264
    action()
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 4
    from random import shuffle,STYLE
'''},
{'date': 'Mon Sep 09 2019 00:20:04.247 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 4
    from random import shuffle, STYLE
ImportError: cannot import name 'STYLE'

ImportError
Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 264
    action()
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 4
    from random import shuffle, STYLE
'''},
{'date': 'Mon Sep 09 2019 00:21:01.300 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 4
    from random import shuffle , STYLE
ImportError: cannot import name 'STYLE'

ImportError
Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 264
    action()
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 4
    from random import shuffle , STYLE
'''},
{'date': 'Mon Sep 09 2019 00:23:59.867 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 6
  IMAGENS = ["CENA", "ARTEFATO" , "FOGO" , "DESMORONAMENTO" , "COBRA" , "PEDRINHAS" , "ARANHAS" , "MUMIA"]*5 STYLE["Width"]=600 
                                                                                                              ^
SyntaxError: invalid syntax
'''},
{'date': 'Mon Sep 09 2019 00:25:00.516 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 6
  IMAGENS = ["CENA" , "ARTEFATO" , "FOGO" , "DESMORONAMENTO" , "COBRA" , "PEDRINHAS" , "ARANHAS" , "MUMIA"]*5 STYLE["Width"]=600 
                                                                                                               ^
SyntaxError: invalid syntax
'''},
{'date': 'Mon Sep 09 2019 00:26:52.326 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 5
  IMAGENS = ["CENA" , "ARTEFATO" , "FOGO" , "DESMORONAMENTO" , "COBRA" , "PEDRINHAS" , "ARANHAS" , "MUMIA"]*5 STYLE["Width"]=600 
                                                                                                               ^
SyntaxError: invalid syntax
'''},
{'date': 'Mon Sep 09 2019 00:29:50.197 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 6
  shuffle(IMAGENS) STYLE["Width"]=600
                    ^
SyntaxError: invalid syntax
'''},
{'date': 'Mon Sep 09 2019 00:30:58.198 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
Exception: <TypeError: Cannot read property 'charAt' of undefined>
'''},
{'date': 'Tue Sep 10 2019 09:49:00.637 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 31
  sala do tesouro = Cena(DI["SALA DO TESOURO"])
        ^
SyntaxError: invalid syntax
'''},
{'date': 'Tue Sep 10 2019 09:53:21.867 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 31
  sala do tesouro = Cena(DI["SALA DO TESOURO"])
        ^
SyntaxError: invalid syntax
'''},
{'date': 'Tue Sep 10 2019 09:54:34.669 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 31
  sala do tesouro = Cena(DI["SALA DO TESOURO"])
        ^
SyntaxError: invalid syntax
'''},
{'date': 'Tue Sep 10 2019 10:02:43.848 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 34
  self.cartas  [Cena(DI[uma_imagem]) for uma_imagem in IMAGEM
                                      ^
SyntaxError: invalid syntax
'''},
{'date': 'Tue Sep 10 2019 10:04:10.955 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 34
  self.cartas [Cena(DI[uma_imagem]) for uma_imagem in IMAGEM
                                     ^
SyntaxError: invalid syntax
'''},
{'date': 'Tue Sep 10 2019 10:05:27.844 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 34
  self.cartas [Cena(DI[uma_imagem])  for uma_imagem in IMAGEM
                                      ^
SyntaxError: invalid syntax
'''},
{'date': 'Tue Sep 10 2019 10:06:30.521 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 34
  self.cartas  [Cena(DI[uma_imagem])  for uma_imagem in IMAGEM
                                       ^
SyntaxError: invalid syntax
'''},
{'date': 'Tue Sep 10 2019 10:07:31.251 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 34
  self.cartas  [Cena(DI[uma_imagem])  for  uma_imagem in IMAGEM
                                       ^
SyntaxError: invalid syntax
'''},
{'date': 'Tue Sep 10 2019 10:28:50.477 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 46
  self.cartas = [Perigo(DI[uma_imagem], uma_imagem) 
                                                                                                                                              ^
SyntaxError: invalid syntax
'''},
{'date': 'Tue Sep 10 2019 10:30:26.627 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 46
  self.cartas = [Perigo(DI[uma_imagem], uma_imagem) 
                                                                                                                                              ^
SyntaxError: invalid syntax
'''},
{'date': 'Tue Sep 10 2019 10:31:15.342 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 46
  self.cartas = [Perigo(DI[uma_imagem], uma_imagem) 
                                                                                                                                              ^
SyntaxError: invalid syntax
'''},
{'date': 'Tue Sep 10 2019 10:36:09.17 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 46
  self.cartas = [Perigo(DI[uma_imagem], uma_imagem) 
                                                                                                                                              ^
SyntaxError: invalid syntax
'''},
{'date': 'Tue Sep 10 2019 10:45:34.798 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 46
  self.cartas = [Perigo(DI[uma_imagem], uma_imagem) 
                                                                                                                                               ^
SyntaxError: invalid syntax
'''},
{'date': 'Tue Sep 10 2019 10:46:37.588 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 46
  self.cartas = [Perigo(DI[uma_imagem], uma_imagem) 
                                                                                                                                               ^
SyntaxError: invalid syntax
'''},
{'date': 'Tue Sep 10 2019 10:47:34.555 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 46
  self.cartas = [Perigo(DI[uma_imagem], uma_imagem) 
                                                                                                                                              ^
SyntaxError: invalid syntax
'''},
{'date': 'Tue Sep 10 2019 10:48:54.93 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 46
  self.cartas = [PERIGO(DI[uma_imagem], uma_imagem) 
                                                                                                                                              ^
SyntaxError: invalid syntax
'''},
{'date': 'Tue Sep 10 2019 10:51:47.185 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 65
    inca = Jogo()  # Inca()
  module <module> line 58
    self.baralho = Carta().baralho()
NameError: name 'Carta' is not defined
'''},
{'date': 'Tue Sep 10 2019 10:53:06.623 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 65
    inca = Jogo()  # Inca()
  module <module> line 58
    self.baralho = Carta().baralho()
NameError: name 'Carta' is not defined
'''},
{'date': 'Tue Sep 10 2019 10:53:34.601 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 65
    inca = Jogo()  # Inca()
  module <module> line 58
    self.baralho = Carta().baralho()
NameError: name 'Carta' is not defined
'''},
{'date': 'Tue Sep 10 2019 10:59:33.377 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 66
    inca = Jogo()  # Inca()
  module <module> line 59
    self.baralho = Carta().baralho()
NameError: name 'Carta' is not defined
'''},